src/Hook.php

```diff
@@ -134,4 +134,41 @@ private static function getDefaultHooks()
            'update',
         ]);
     }
+
+    /**
+     * Check if a hook is valid
+     * @param string $dir
+     * @param string $hook
+     * @return bool
+     */
+    private static function isCustomHook($dir, $hook)
+    {
+        return in_array($hook, self::getCustomHooks($dir));
+    }
+
+    /**
+     * Get custom hooks from config `custom-hooks` section.
+     * @param string $dir
+     * @return array
+     */
+    public static function getCustomHooks($dir)
+    {
+        $customHooks = self::getConfig($dir, 'custom-hooks');
+
+        if (! $customHooks) {
+            return [];
+        }
+
+        if (! is_array($customHooks)) {
+            throw new Exception('Custom hooks must be an array.');
+        }
+
+        $configIndex = array_search('config', $customHooks);
+
+        if ($configIndex !== false) {
+            unset($customHooks[$configIndex]);
+        }
```

Comment:
I'm not sure I understand why we have this

Reply:
I think that `config` should not be used as a hook. You're thought on this?

Reply:
Oh that's right! Makes sense. Could you make a note of this in the readme section for custom hooks that `config` is not allowed